### PR TITLE
add Checkbox docs to site

### LINKED ISSUE #2194: Check Box Documentation

### PR DESCRIPTION
**Pull request by feliciackh**
Closes #2194

### CODE REVIEW

File: site/docs/components/checkbox/accessibility.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  label: Accessibility
```
Comment (alycrys) [inline site/docs/components/checkbox/accessibility.mdx:7]: exclude: true

File: site/docs/components/checkbox/examples.mdx [hunk 2]
```diff
@@ -0,0 +1,138 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  label: Examples
```
Comment (alycrys) [inline site/docs/components/checkbox/examples.mdx:7]: exclude: true

File: site/docs/components/checkbox/examples.mdx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+<LivePreview componentName="checkbox" exampleName="Default">
+
```
Comment (alycrys) [inline site/docs/components/checkbox/examples.mdx:14]: add title

File: site/docs/components/checkbox/examples.mdx [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+</LivePreview>
+
+<LivePreview componentName="checkbox" exampleName="VerticalGroup" >
+
+### Checkbox group: vertical (default)
+
+Checkbox Group allows selection of one or more values from a given set of choices. They can be aligned horizontally or vertically. By default, the group is aligned vertically.
+
+Guidance:
```
Comment (alycrys) [inline site/docs/components/checkbox/examples.mdx:25]: should this be a heading?

File: site/docs/components/checkbox/examples.mdx [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+<LivePreviewControls>
+<LivePreview componentName="checkbox" exampleName="Default">
+
+A single checkbox can be either selected or deselected.
+
+</LivePreview>
+
+<LivePreview componentName="checkbox" exampleName="VerticalGroup" >
+
+### Checkbox group: vertical (default)
```
Comment (alycrys) [inline site/docs/components/checkbox/examples.mdx:21]: Checkbox Group

File: site/docs/components/checkbox/accessibility.mdx (hunk 1 above)
Comment (feliciackh) [inline site/docs/components/checkbox/accessibility.mdx:7]:
```suggestion
  label: Accessibility
  exclude: true
```

File: site/docs/components/checkbox/usage.mdx [hunk 6]
```diff
@@ -0,0 +1,44 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  label: Usage
```
Comment (alycrys) [inline site/docs/components/checkbox/usage.mdx:7]: exclude: true

File: site/src/examples/checkbox/ControlledGroup.tsx [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+      value: "alternatives",
+    },
+    {
+      label: "Equities",
+      value: "equities",
+    },
+    {
+      disabled: true,
+      label: "Fixed income",
+      value: "Fixed income",
```
Comment (alycrys) [inline site/src/examples/checkbox/ControlledGroup.tsx:17]: value: "fixed-income" maybe?

Comment (feliciackh) [reply site/src/examples/checkbox/ControlledGroup.tsx:17]: oops this example shouldnt be here - removing all controlled related content

File: site/docs/components/checkbox/accessibility.mdx [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
```
Comment (alycrys) [inline site/docs/components/checkbox/accessibility.mdx:7]: no need for label if you exclude. At least all the components have it removed

File: site/docs/components/checkbox/usage.mdx [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+import { Checkbox, CheckboxGroup } from "@salt-ds/core";
+```
+
+### Props
+
+{/* Update packageName and componentName below as needed */}
+{/* packageName is optional and defaults to "core" if omitted */}
+
+<PropsTable packageName="core" componentName="Checkbox" />
+
```
Comment (alycrys) [inline site/docs/components/checkbox/usage.mdx:41]: should you add props for CheckboxGroup too?

Comment (feliciackh) [reply site/docs/components/checkbox/usage.mdx:41]: great spot - have added

**Review by alycrys — APPROVED** [review]

**Review by lilyvc — APPROVED** [review]

File: site/docs/components/checkbox/examples.mdx (hunk 4 above)
Comment (feliciackh) [reply site/docs/components/checkbox/examples.mdx:27]: keeping it as is for now

**Pull request merged**